docs/sources/flow/reference/components/loki.process.md

```diff
@@ -0,0 +1,191 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.process
+title: loki.process
+---
+
+# loki.process
+
+`loki.process` receives log entries from other loki components, applies one or
+more processing _stages_, and forwards the results to the list of receivers
+in the component's arguments.
+
+The `loki.process` component is a multi-purpose tool that can parse, transform,
+and filter log entries before they're passed to a downstream component. The
```

Comment:
This feels like a second intro paragraph to the component, can this be reworded?